msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -183,22 +183,21 @@ items:
 
        You can see the following page:
 
-         :::image type="content" source="~/assets/images/sbs-notification-bot/default-program.png" alt-text="Default Program" lightbox="~/assets/images/sbs-notification-bot/default-program-1.png":::
-
-    1. Use the following code after `for (const target of await bot.notification.installations()) { ` in Visual Studio Code.
-
-         ```bash
-         if (target.type === "Channel") {
-           const members = await target.members();
-           await target.sendMessage("This is a message to channel: " + members.length + " members.");
-           for (const member of members) {
-             await member.sendMessage("This is a message to Member: " + member.account.email);
-           }
-         }
-         /* 
-         ```  
-
-          :::image type="content" source="~/assets/images/sbs-notification-bot/code-add.png" alt-text="Code Add" lightbox="~/assets/images/sbs-notification-bot/code-add-1.png":::
+         :::image type="content" source="~/assets/images/sbs-notification-bot/default-program.png" alt-text="Default Program" lightbox="~/assets/images/sbs-notification-bot/default-program.png":::
+
+    1. Use the following code after `for (const target of installations) {` in Visual Studio Code.
```

Comment:
Why a period at the end of the sentence? It should be : as we are using the word following